### PR TITLE
Msue 156 support score percentiles in sift dotnet

### PR DESCRIPTION
**Pull request by binish-exalture**
## Checklist
- [x] The change was thoroughly tested manually
- [x] The change was covered with unit tests
- [x] The change was tested with real API calls (if applicable)
- [x] Necessary changes were made in the integration tests (if applicable)
- [x] New functionality is reflected in README

### CODE REVIEW

File: Test/Test.cs [hunk 1]
```diff
@@ -1316,6 +1316,33 @@ public void TestLoginEvent()
...
+                transaction_type = "$sale",
+                transaction_status = "$failure",
+                decline_category = "$invalid"
+            };
+
+            EventRequest eventRequest = new EventRequest
+            {
+                Event = transaction,
+                AbuseTypes = { "legacy", "payment_abuse" },
+                Fields = { "SCORE_PERCENTILES" },
```
Comment (viaskal-sift) [inline Test/Test.cs:1340]: this if kind of revealing the implementation details. No need for the customer to specify GET-params, this is what happening under the hood. Would be better to provide something like `IncludeScorePercentiles = TRUE` or something like this. Good example is from sift-java - https://github.com/SiftScience/sift-java/commit/6e2781c9f9c80d1bf5b373869be24ecc5b6983f1#diff-9582a6fb9ef5c3c54f41686a60ccead81902999fed7b92f0de1e4da753630b9cR96

Comment (binish-exalture) [reply Test/Test.cs]: Done with the said changes

Comment (binish-exalture) [reply Test/Test.cs]: Resolved

File: README.md [hunk 2]
```diff
@@ -50,6 +50,22 @@ The official Sift .NET client, supporting .NET Standard 2.0+
...
+            get
+            {
+                var url = new Uri(EventsUrl);
+                if (IncludeScorePercentile)
+                {
+                    url = url.AddQuery("fields", "SCORE_PERCENTILES");
+                }
+            }
+     }
+
```
Comment (viaskal-sift) [inline README.md:68]: No need to reflect this in README, instead - let's just provide a comment (or example) how we should modify the `EventRequest` in order to support score percentiles

**Review by haneeshv — APPROVED** [review]
It seems fine.

**Pull request merged**